game/missiongenerator/frontlineconflictdescription.py

```diff
@@ -7,15 +7,15 @@
 from dcs.mapping import Point
 from shapely.geometry import LineString, Point as ShapelyPoint
 
+from game.settings import Settings
 from game.theater.conflicttheater import ConflictTheater, FrontLine
 from game.theater.controlpoint import ControlPoint
 from game.utils import Heading
 
 
-FRONTLINE_LENGTH = 80000
-
-
 class FrontLineConflictDescription:
+    max_frontline_length: int
```

Comment:
where is this used? i can only find the setter in `frontline_position()`. You are retrieving this property always from the settings class and not fom the `FrontLineConflictDescription` if i get it right

Reply:
Good find! It's actually not used anymore. I had originally intended that it would be set in frontline_position() and then could be accessed locally, but eventually refactored so the value from the settings class is used everywhere. I had forgotten to remove it.

Reply:
It is now removed as it's not needed.